src/utils/formHelperFunctions.js

```diff
@@ -248,7 +244,7 @@ function getFieldError(value, servar, signatureRef) {
       noVal = !value && servar.metadata?.must_check;
       break;
     case 'signature':
-      noVal = signatureRef[servar.key].isEmpty();
+      noVal = value.isEmpty();
```

Comment:
isEmpty used to be called on the ref element directly, i'm not sure it can be called on value like this. probably also worth testing